README.md

```diff
@@ -63,7 +63,7 @@
     
         private static void index(Context ctx, ConnectionPool connectionPool)
         {
-            ctx.render("/projectname/index.html");
+            ctx.render("/projectname/startPage.html");
```

Comment:
please keep using the index as it will otherwise break the application for all other teams

Reply:
renamed to index.html